apps/i18n/dance/en_us.json

```diff
@@ -27,5 +27,6 @@
   "danceFeedbackUseSetSize": "Use the `set backup_dancer2 size` block to make that dancer smaller.",
   "danceFeedbackUseSetTint": "Use the `set tint` block to change a dancer's tint.",
   "danceFeedbackUseStartMapping": "Try adding the <xml><block type=\"Dancelab_startMapping\"><title name=\"SPRITE\">right_unicorn</title><title name=\"PROPERTY\">\"scale\"</title><title name=\"RANGE\">\"bass\"</title></block></xml> block to your program.",
+  "danceFeedbackKeepCoding": "Keep coding! Something's not quite right yet.",
```

Comment:
Can we just use the existing string from common/en_us.json? Or is there a chance we'd want to make this specific for dance?

Reply:
I added here because of how the strings are loaded - `danceMsg` is imported from [here](https://github.com/code-dot-org/code-dot-org/blob/a5363cbe6d4919fffec983ddef1b42c3bb2c5e81/apps/src/dance/Dance.js#L16) and strings are loaded [here](https://github.com/code-dot-org/code-dot-org/blob/61d93eec92ae839e13fd0a867face0f82a048701/apps/src/dance/locale.js#L5) from `dance` so added string in this file.
And it's also nice to keep it separate in case curriculum folks want to customize. Thanks!